### DISCUSSION TITLE
Listing repositories a user has access.

### DISCUSSION THREAD

**Discussion by llrs** [community/community#45236] [Repositories]
### Select Topic Area

Product Feedback

### Body

The only way to find repositories a user has access is visiting [`https://github.com/settings/repositories`](https://github.com/settings/repositories) but it is not possible to search in the list (beyond what the browser allows) or to hide repositories shared from a user or organization.

I tried with `https://github.com/<user>?tab=repositories&type=source` but there the search isn't limited to the ones I have access and it doesn't seem possible to use repo: or org: to exclude the user. 

In addition, by the number of [votes in SO](https://stackoverflow.com/questions/12453358/see-repositories-where-i-am-collaborator) it seems like this webpage is not easy to find.

**Comment by airtower-luna** [comment]
Using the API might help: https://docs.github.com/en/rest/repos/repos?apiVersion=2022-11-28#list-repositories-for-the-authenticated-user

**Reply by llrs** [reply]
Thanks might help, but only if the user wants to delve into that. In this one usage case I think it would be better to have some user interface to do it.
